### PR TITLE
Change "Ban" to "Ban from room" to fix an inconsistency in the Admin Tools dialog

### PR DESCRIPTION
**Pull request by EECvision**
Fixes https://github.com/vector-im/element-web/issues/20578

I have changed the Admin tools button "Ban" to "Ban from room" so as to be consistently phrased with Remove from room

Signed-off-by: Emmanuel Ezeka eec.studies@gmail.com

---
This PR currently has no changelog labels, so will not be included in changelogs.

A reviewer can add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is, or add `Type: [enhancement/defect/task]` to the description and I'll add them for you.

Preview: https://pr8281--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

**Comment by EECvision** [comment]
Hi @t3chguy, please can you tell me why I am getting the error?

**Review by turt2live — CHANGES_REQUESTED** [review]
You've changed the string, but not the source file. Additionally, as noted in https://github.com/matrix-org/matrix-react-sdk/pull/7576, this needs to be space-aware.

The PR description should also not have the "Fixes $link" in a code block because it prevents github from working. The PR title should also be a sentence.

**Comment by EECvision** [comment]
> You've changed the string, but not the source file. Additionally, as noted in #7576, this needs to be space-aware.
> 
> The PR description should also not have the "Fixes $link" in a code block because it prevents github from working. The PR title should also be a sentence.

Alright, thank you. I will fix that.

File: src/components/views/right_panel/UserInfo.tsx [hunk 1]
```diff
@@ -653,7 +653,7 @@ const BanToggleButton = ({ room, member, startUpdating, stopUpdating }: Omit<IBa
             room.isSpaceRoom() ? ConfirmSpaceUserActionDialog : ConfirmUserActionDialog,
             {
                 member,
-                action: isBanned ? _t("Unban") : _t("Ban"),
+                action: isBanned ? _t("Unban") : _t("Ban from room"),
```
Comment (t3chguy) [inline src/components/views/right_panel/UserInfo.tsx:656]: Surely now `ban` is inconsistent from `unban`?

File: src/components/views/right_panel/UserInfo.tsx [hunk 2]
```diff
@@ -719,7 +719,7 @@ const BanToggleButton = ({ room, member, startUpdating, stopUpdating }: Omit<IBa
         });
     };
 
-    let label = _t("Ban");
+    let label = _t("Ban from room");
     if (isBanned) {
         label = _t("Unban");
```
Comment (t3chguy) [inline src/components/views/right_panel/UserInfo.tsx:724]: Same as above

File: src/components/views/right_panel/UserInfo.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (EECvision) [reply src/components/views/right_panel/UserInfo.tsx:656]: Okay, I am towards fixing it.

**Comment by turt2live** [comment]
Please do not force push after review has been given. It makes review significantly harder and less likely to happen :(

File: src/components/views/right_panel/UserInfo.tsx [hunk 3]
```diff
@@ -653,7 +653,9 @@ const BanToggleButton = ({ room, member, startUpdating, stopUpdating }: Omit<IBa
             room.isSpaceRoom() ? ConfirmSpaceUserActionDialog : ConfirmUserActionDialog,
             {
                 member,
-                action: isBanned ? _t("Unban") : _t("Ban"),
+                action: room.isSpaceRoom() ?
+                    isBanned ? _t("Unban from space") : _t("Ban from space")
+                    : isBanned ? _t("Unban from room") : _t("Ban from room"),
```
Comment (t3chguy) [inline src/components/views/right_panel/UserInfo.tsx:658]:
```suggestion
                action: room.isSpaceRoom()
                    ? (isBanned ? _t("Unban from space") : _t("Ban from space"))
                    : (isBanned ? _t("Unban from room") : _t("Ban from room")),
```

File: src/components/views/right_panel/UserInfo.tsx [hunk 4]
```diff
@@ -1463,8 +1469,8 @@ interface IProps {
     user: Member;
     room?: Room;
     phase: RightPanelPhases.RoomMemberInfo
-        | RightPanelPhases.SpaceMemberInfo
-        | RightPanelPhases.EncryptionPanel;
+    | RightPanelPhases.SpaceMemberInfo
+    | RightPanelPhases.EncryptionPanel;
```
Comment (t3chguy) [inline src/components/views/right_panel/UserInfo.tsx:1473]:
```suggestion
        | RightPanelPhases.SpaceMemberInfo
        | RightPanelPhases.EncryptionPanel;
```

Comment (t3chguy) [reply src/components/views/right_panel/UserInfo.tsx:1473]: Revert the formatting you did, it is unrelated to your PR

**Comment by EECvision** [comment]
Hi @t3chguy, I have committed your suggestions. Is there any other change I need to make in the PR?

**Review by turt2live — APPROVED** [review]
Thanks!

**Comment by EECvision** [comment]
> Thanks!



> Thanks!

Thank you

**Pull request merged**